server/routes/courtRegister/amendCourtDetailsView.ts

```diff
@@ -0,0 +1,53 @@
+import type { AmendCourtDetailsForm } from 'forms'
+import { Court, CourtType } from '../../@types/courtRegister'
+
+interface SelectItem {
+  value: string
+  text: string
+  selected: boolean
+}
+
+function sortAlphabetically(courtType1: SelectItem, courtType2: SelectItem) {
+  if (courtType1.text < courtType2.text) {
```

Comment:
could you simplify this whole block to `courtType1.text - courtType2.text`?

Reply:
can I????  Never seen a "subtract" operator on a string

Reply:
not sure - you're doing greater than or less than here anyway so maybe :shrug:

Reply:
sadly not, looks like `<` is ok on strings but not `-` - would been a nice idea though